lib/service/storage/elasticsearch.js

```diff
@@ -3154,14 +3155,37 @@ class ElasticSearch extends Service {
           },
         },
         index: await this._getAvailableIndice(index, HIDDEN_COLLECTION),
-      });
+        wait_for_active_shards: await this._getWaitForActiveShards(),
+      };
+
+      await this._client.indices.create(esRequest);
```

Comment:
This also need to be done when updating a collection `updateCollection`

Reply:
I think you were thinking about `collection:truncate` because `collection:updateCollection` do not create a new Elasticsearch index. I'm going to update `collection:truncate` according to your suggestion 👍

Reply:
Ok I didn't realize it was only for creation and not for mappings update. But good catch for `truncate` ;-)